src/notifications.js

```diff
@@ -122,7 +123,21 @@ export default class Notification {
       combineTopics(prefix, postFixes);
     });
 
-    return combinedTopics.concat(precombinedTopics);
+    return this.truncateTopicList(combinedTopics.concat(precombinedTopics));
+  }
+
+  truncateTopicList (topics) {
+    const keptTopics = topics.slice(0, MAX_SUBSCRIBABLE_TOPICS);
+    if (topics.length > MAX_SUBSCRIBABLE_TOPICS) {
+      let droppedTopics = topics.slice(MAX_SUBSCRIBABLE_TOPICS);
+      if (droppedTopics.length > 20) {
```

Comment:
Can we get a var for this please? Something like `DROPPED_TOPICS_DISPLAY_COUNT` or something along those lines.